draft-ietf-teas-5g-ns-ip-mpls.md

```diff
@@ -1370,13 +1370,13 @@ to TN QoS Classes may be rather common.
    only occur in traffic that was de-prioritized by ingress admission control {{sec-inbound-edge-resource-control}} or in non-premium (best-effort) classes.  Capacity planning and management, as described in {{sec-capacity-planning}}, ensures that enough
    capacity is available to fulfill all approved slice requests.
 
-#  Transport Planes Mapping Models
+#  Transport Planes Mapping Models {#transport-plane-mapping-models}
 
    A network operator can define multiple transport planes. A transport plane may be realized in multiple ways such as (but not limited to):
 
    * A mesh of RSVP-TE {{?RFC3209}} or SR-TE {{?RFC9256}} tunnels created with specific optimization criteria and
    constraints. For example, mesh "A" might represent tunnels optimized for latency, and mesh "B" might represent tunnels optimized for high capacity.
-   * A flex-algo {{?RFC9350}} with a particular metric-type (e.g., latency).
+   * A flex-algo {{?RFC9350}} with a particular metric-type (e.g., latency), or one that only uses links with particular properties (e.g. MACsec link), or excludes links that are within a particular geography.
```

Comment:
```suggestion
   * A flex-algo {{?RFC9350}} with a particular metric-type (e.g., latency), or one that only uses links with particular properties (e.g., MACsec links), or excludes links that are within a particular geography.
```

I would add a new ref entry for MACsec: 

IEEE, "802.1AE: MAC Security (MACsec)", https://1.ieee802.org/security/802-1ae/

Reply:
The ref is now added: [e52ce33](https://github.com/boucadair/5g-slice-realization/pull/137/commits/e52ce33136c606a306c2652c2363d1399afbe4df)